storage/innobase/handler/ha_innodb.cc

```diff
@@ -19672,6 +19692,10 @@ static MYSQL_SYSVAR_BOOL(force_primary_key,
   "Do not allow creating a table without primary key (off by default)",
   NULL, NULL, FALSE);
 
+static MYSQL_SYSVAR_BOOL(alter_copy_bulk, innodb_alter_copy_bulk,
+  PLUGIN_VAR_NOCMDARG,
+  "Allow bulk insert operation for copy alter operation", NULL, NULL, TRUE);
```

Comment:
As far as I can tell, there is no regression test that would enable this setting. It would be useful to cover both normal and partitioned tables, both small ones (merge sorting happens in memory) and larger ones (temporary files will be used). It would also be good to cover `ALTER IGNORE TABLE` and operations that involve `FOREIGN KEY`.